### DISCUSSION TITLE
Limit Framerate

### DISCUSSION THREAD

**Discussion by cursorweb** [replit/kaboom#296] [Ideas]
When in multiplayer games, its very useful to limit framerate so you don't send too many data packets at any given time.

Another use is just to make games faster.

The syntax looks like this:
```js
kaboom({
  frameRate: 60,
});
```

**Comment by lajbel** [comment]
Cool idea!

**Comment by slmjkdbtl** [comment]
I like that 👍

**Comment by yoannes** [comment]
Hi,

Is there any way to get this done? it's a nice feature

**Comment by slmjkdbtl** [comment]
Implemented in d59b9d3

**Comment by cursorweb** [comment]
yay!!
~~only took 2 years~~

**Comment by lajbel** [comment]
I love hype
